test/functional/Java21andUp/build.xml

```diff
@@ -77,7 +77,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<target name="build">
 		<if>
 			<not>
-				<matches string="${JDK_VERSION}" pattern="^(8|9|1[0-9]|20)$$" />
+				<matches string="${JDK_VERSION}" pattern="^(8|9|1[0-9]|20|21)$$" />
```

Comment:
This is intended to be temporary, right? I think it deserves a comment to that effect.

Reply:
Add comment there as the test suites are disabled for the moment and will be enabled once we update the related code to support the latest APIs in JDK21.